frontend/components/layout/HeaderLogo.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+    <div class="flex items-center">
+        <nuxt-link to="/" class="hidden md:block text-gray-700 hover:text-gray-600 no-underline">
+            <branding-icon class="h-12 w-12" />
+        </nuxt-link>
+        <Select
+            :options="options"
+            :value="pageName"
+            title="Unified Auction"
+            :is-title-click-able="true"
+            :show-selected-desktop="true"
+            class="md:ml-2 mb-0"
+            @input="switchPage"
+        >
+            <template #text-prefix>
+                <branding-icon class="md:hidden h-12 w-12" />
+            </template>
+        </Select>
+    </div>
+</template>
+
+<script lang="ts">
+import Vue from 'vue';
+import Select from '../common/inputs/Select.vue';
+import { generateLink } from '../../helpers/generateLink';
+import BrandingIcon from '~/assets/icons/logo.svg';
+
+export default Vue.extend({
+    name: 'HeaderLogo',
+    components: { Select, BrandingIcon },
+    props: {
+        network: {
+            type: String,
+            default: undefined,
+        },
+        pageName: {
+            type: String,
+            default: undefined,
+        },
+    },
+    computed: {
+        options() {
+            return [
+                { label: 'Maker Auction Services', value: '' },
+                { label: 'Collateral auctions', value: 'collateral' },
+                { label: 'Surplus auctions', value: 'surplus' },
+            ];
+        },
+    },
+    methods: {
+        switchPage(page: string) {
+            if (this.$router) {
+                const link = generateLink(this.network, page);
+                this.$router.push(link);
```

Comment:
I personally always use control-click and dislike all non-native links on websites. For example, how would the user be able to open another auction type in a separate tab? I know it's much more work, but do you see a way of making it work without reimplementing the whole select? Providing `href` key to the options?

Reply:
I was debating this as well, as I generally agree (also helps with SEO optimisation). 

Would passing a `href` be enough? I would have gone with using `nuxt-link`. Are there any big differences between the two? I just know that the NextJS Link component does a lot more then just redirecting.

Reply:
No, I mean, maybe you extend Select component which is quite complicated component (with dropdown vs modal logic) by adding new `{ label: 'Maker Auction Services', value: '', href: '/' }` so when it's present it turns into `nuxt-link` instead of a button?

Reply:
Ahhh, that makes more sense now 🤗 

I think it is worth it, Ill get on it.